setuptools/config/_apply_pyprojecttoml.py

```diff
@@ -195,8 +195,10 @@ def _python_requires(dist: "Distribution", val: dict, _root_dir):
 
 
 def _dependencies(dist: "Distribution", val: list, _root_dir):
-    existing = getattr(dist, "install_requires", [])
-    _set_config(dist, "install_requires", existing + val)
```

Comment:
Unless I'm reading PEP 621 wrong, couldn't `dependencies` also be one of the keys that can be specified as `dynamic`? Wouldn't this always overwrite?

Reply:
Ah yep, all good; wasn't looking at the bigger picture 😅